lib/src/modules/settings/scenes/fvm_settings.scene.dart

```diff
@@ -37,9 +38,9 @@ class FvmSettingsScene extends StatelessWidget {
           ),
           const Divider(),
           SwitchListTile(
-            title: const Text('Skip setup Flutter on install'),
-            subtitle: const Text('This will only clone Flutter and not install'
```

Comment:
The subtitle is just one string, it was added the comma for legibility in the code but can be put together in the same string value for translation